src/main/java/com/sproutsocial/nsq/NSQMessage.java

```diff
@@ -63,4 +65,16 @@ public void touch() {
         connection.touch(id);
     }
 
+    @Override
+    public void forceFlush() {
```

Comment:
What are the semantics of this method.
It looks like it is doing an operation on connection and is not scoped to the current message.
Is connection/subscriber a better place for this?

Reply:
We don't have direct access to the connection from the consumer side.  

This is being added to support strongly consistent finishes.  This speeds up tests and makes them deterministic.